enterprise/admin/forms.py

```diff
@@ -501,3 +502,43 @@ class Meta:
             'decrypted_password': forms.widgets.PasswordInput(),
             'decrypted_sftp_password': forms.widgets.PasswordInput(),
         }
+
+
+class TransmitEnterpriseCoursesForm(forms.Form):
+    """
+    Form to transmit courses metadata for enterprise customers.
+    """
+    catalog_user = forms.CharField(
+        label=_('Enter enterprise worker username.'),
```

Comment:
is enterprise worker user not confusing?

Reply:
I will change it to `enterprise channel worker` as setup here: https://github.com/edx/edx-internal/blob/master/ansible/vars/stage-edx.yml